sdk/Dagger.SDK.Mod/Dagger.SDK.Mod.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If the source generator plan is going well, we don't need this package anymore. 🎉